numpy/distutils/exec_command.py

```diff
@@ -427,11 +421,6 @@ def test_execute_in(**kws):
     os.remove(tmpfile)
     print ('ok')
 
-def test_svn(**kws):
-    s, o = exec_command(['svn', 'status'],**kws)
-    assert s, (s, o)
-    print ('svn ok')
-
 def test_cl(**kws):
```

Comment:
no these tests are not run, probably should be moved to the more proper testfile.